### PR TITLE
Export a selection of post types with `wp export`

### PR DESCRIPTION
**Pull request by stevegrunwell**
Enables the `--post_type` argument of `wp export` to accept a comma-separated list of post types, rather than forcing the user to run the exporter multiple times if they want just a subset of the post types available.

### CODE REVIEW

File: php/commands/export.php [hunk 1]
```diff
@@ -179,10 +180,18 @@ private function check_post_type( $post_type ) {
...
+		$post_type = array_unique( array_filter( explode( ',', $post_type ) ) );
 		$post_types = get_post_types();
-		if ( !in_array( $post_type, $post_types ) ) {
-			WP_CLI::warning( sprintf( 'The post type %s does not exist. Choose "all" or any of these existing post types instead: %s', $post_type, implode( ", ", $post_types ) ) );
-			return false;
+
+		foreach ( $post_type as $type ) {
+			if ( ! in_array( $type, $post_types ) ) {
+				WP_CLI::warning( sprintf(
+					'The post type %s does not exist. Choose "all" or any of these existing post types instead: %s',
```
Comment (danielbachhuber) [inline php/commands/export.php]: I think this syntax needs to be `any` instead of `all`. And, if I provide `any`, it should skip this check.

**Comment by danielbachhuber** [comment]
Seems reasonable. Thanks for following the existing CSV pattern. Can you update the tests so they pass on CI?

File: features/export.feature [hunk 2]
```diff
@@ -74,6 +74,58 @@ Feature: Export content.
       10
       """
 
+  Scenario: Export a comma-separated list of post types
+    Given a WP install
+
+    When I run `wp plugin install wordpress-importer --activate`
+    Then STDERR should not contain:
```
Comment (danielbachhuber) [inline features/export.feature]: You should always test what you're expecting — `Then STDOUT should contain.. Success:`

File: features/export.feature [hunk 3]
```diff
@@ -74,6 +74,58 @@ Feature: Export content.
...
+    And I run `wp post generate --post_type=page --count=10`
+    And I run `wp post generate --post_type=post --count=10`
+    And I run `wp post generate --post_type=attachment --count=10`
+    And I run `wp post list --post_type=page,post,attachment --format=count`
+    Then STDOUT should be:
+      """
+      30
+      """
+
+    When I run `wp export --post_type=page,post`
```
Comment (danielbachhuber) [inline features/export.feature]: Good re: testing with a subset.

**Comment by stevegrunwell** [comment]
Hey Daniel,

I apologize that it took me so long to get back to this, but the PR should be in a better state for consideration. Thank you for your feedback and guidance, WP-CLI was my first attempt at BDD via Behat.

**Comment by danielbachhuber** [comment]
Cool, thanks @stevegrunwell. I'll land after #1990

**Pull request merged**